jira-feature-collector/src/main/java/com/capitalone/dashboard/datafactory/jira/JiraDataFactoryImpl.java

```diff
@@ -254,8 +255,20 @@ public String setQuery(String query) {
 				}
 			} catch (Exception e) {
```

Comment:
@amitmawkin :  I just re-checked my statement above to get the odd exception behavior - upon retrospective, I am, in fact, able to catch the exception I thought previously uncatchable:

```java
com.atlassian.jira.rest.client.api.RestClientException
```

I will add this fix to my next commit on this PR and fix.  If we do see an issue going forward where an uncatchable exception is being thrown, i will follow up with Jira support - I hadn't thought of that originally.